src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/item/spellbook/SpellBookMenu.java

```diff
@@ -0,0 +1,51 @@
+package com.github.minecraftschurlimods.arsmagicalegacy.common.item.spellbook;
+
+import com.github.minecraftschurlimods.arsmagicalegacy.api.ArsMagicaAPI;
+import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMMenuTypes;
+import net.minecraft.world.Container;
+import net.minecraft.world.SimpleContainer;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+
+public class SpellBookMenu extends AbstractContainerMenu {
```

Comment:
Naming seems odd here, why did you use k and j1 instead of j? Also, I don't find the comments necessary, most of the code is self-explanatory.